### PR TITLE
fix: only show delete image button when editing

### PR DESCRIPTION
**Pull request by max-nextcloud**
Signed-off-by: Max <max@nextcloud.com>

* Target version: master 

### Summary
We used to show the delete image button even in read only views - such as a public share without editing permission.
Only show it when people are actually allowed to remove the image.

### CODE REVIEW

File: src/tests/nodes/ImageView.spec.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ describe('Image View src attribute based on markdown', () => {
 	const factory = attrs => {
 		const propsData = {
 			extension: { options: {currentDirectory: '/current'} },
+			editor: { isEditable: true },
```
Comment (vinicius73) [inline src/tests/nodes/ImageView.spec.js:16]: Do we have a test case for when this value is false?

**Review by eneiluj — APPROVED** [review]
Fine by me!

**Pull request merged**